rollup.config.js

```diff
@@ -1,18 +1,12 @@
 import ts from "@wessberg/rollup-plugin-ts";
 import path from "path";
-import commonjs from "rollup-plugin-commonjs";
 import nodeResolve from "rollup-plugin-node-resolve";
-import { uglify } from "rollup-plugin-uglify";
+import { terser } from "rollup-plugin-terser";
+
 const extensions = [".ts", ".tsx"];
 
-function isBareModuleId(id) {
-	if (id.startsWith(".")) {
-		return false;
-	}
-	if (id.startsWith("src")) {
-		return false;
-	}
-	return !id.includes(path.join(process.cwd(), "src"));
+function isExternal(id) {
+  return id.startsWith(".") === false && path.isAbsolute(id) === false;
```

Comment:
Nitpick: Wrong indentation. 

(I should probably add an .editorconfig file.)

Reply:
I can rebase if you will setup prettier. By default my editor enforce 2-space indentation.